src/app/member/page.tsx

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+import { MemberList } from "./components/MemberList";
+const Page = () => {
+  return (
+    <div className="p-8">
+      <MemberList />
```

Comment:
can we use this div into `MemberList` so that all design can be handled in one component

Reply:
Nope we can use this div  in `MemberList` because here `Member Header` will be placed